.env.example

```diff
@@ -0,0 +1,6 @@
+DB_USER=username
```

Comment:
This file should be in .gitignore and should not be pushed to git

Reply:
Why, it is an example env file not real variables. Should it also not be incluclded?

Reply:
.env and other .env variable should not be saved in git. since different .env files could mean different staging environments. For example: `.env.dev`, `.env.qa`, `.env.prod`.  For now we can save this, but make sure to add this file to gitignore for later.